src/parsers/poscar.js

```diff
@@ -55,7 +55,15 @@ function toPoscar(materialOrConfig, omitConstraints = false) {
     ].join('\n');
 }
 
+export function lineCount(poscarFileContent) {
```

Comment:
We should calculate the number of atoms, otherwise, this function doesn't make much sense to include. Let's get back to summing up the values on line 7: https://www.vasp.at/wiki/index.php/POSCAR